text-utils/col.c

```diff
@@ -167,21 +167,21 @@ static inline void col_putchar(wchar_t ch)
  */
 static void flush_blanks(struct col_ctl *ctl)
 {
-	int half = 0, i, nb = ctl->nblank_lines;
+	int half = 0, i;
 
-	if (nb & 1) {
+	if (ctl->nblank_lines & 1) {
 		if (ctl->fine)
 			half = 1;
 		else
-			nb++;
+			ctl->nblank_lines++;
 	}
-	nb /= 2;
-	for (i = nb; --i >= 0;)
+	ctl->nblank_lines /= 2;
```

Comment:
Is it correct? It seems you modify global ctl->nblank_lines, but the original code uses a local variable and it does not refresh the original global setting -- nblank_lines is unmodified by this function in original code.

Reply:
Good point. I removed the commit that had ill thought variable removal.